manifest.json

```diff
@@ -1,13 +1,9 @@
 {
-	"content_scripts": [
-		{
-			"js": ["lib/content-script.js"],
-			"matches": [
-				"https://github.com/*/*/issues/*",
-				"https://github.com/*/*/pull/*"
-			]
-		}
-	],
+	"permissions": ["tabs", "webNavigation", "scripting"],
+	"background": {
+		"service_worker": "lib/background.js"
+	},
+	"host_permissions": ["https://github.com/*"],
```

Comment:
[Question] Why not inject the script on all `github.com/` pages? I like only injecting when we need it, but asking for more permissions seems odd. And the script is pretty small.

Reply:
### **Thank you for challenging me!**
I updated this PR with a solution that keeps permissions the same as before 😊